pkgs/development/tools/misc/binutils/default.nix

```diff
@@ -19,10 +19,11 @@ in
 stdenv.mkDerivation rec {
   name = targetPrefix + basename;
 
-  src = fetchurl {
+  # HACK to ensure that we preserve source from bootstrap binutils to not rebuild LLVM
+  src = stdenv.__bootPackages.binutils-unwrapped.src or (fetchurl {
```

Comment:
This is really gross. Maybe we should just `builtins.fetchurl` everywhere? IMO with `fetchurl`, `builtins.fetchurl`, `fetchurlBoot` and `<nix/fetchurl.nix>`, there are too many ways to fetch in nixpkgs.

Reply:
It's good enough for now though. At least we avoid the mass rebuild. Any improvement can be another mass rebuild.